### ISSUE TITLE
Could this project run according to the provided code？

### ISSUE THREAD

**Issue by jiahuanluo** [henrileleu/covid19#1]
Thanks for your job.

Could this project run according to the provided code？

**Comment by henrileleu** [comment]
Yes, the program should run. It was programmed in VS2019, and might need some tweaking if compiled in a different compiler. 

As mentioned in the ReadMe, you need to have Intel MKL (2020.2.254) and TBB librairies (2020.2.899) also to be able to compile and run the program. They are not essential and could technically be removed and replaced. 

You'll also need the parameters file "param_xx.csv" that is also in the repository to run the simulation.

**Comment by jiahuanluo** [comment]
Thanks!
Finally, I made the code ran after installing the correct package version!

BTW, may I ask what the proper setting for `size` and `n` is when using the `params_nyc.csv` and `params_fr.csv`?

Thanks again!
